eng/pipelines/libraries/prepare-publish-for-tests.yml

```diff
@@ -0,0 +1,55 @@
+# Steps used to prepare the Artifacts Staging Directory with required files for
```

Comment:
I feel like we should rename this file to something like, `prepare-for-bin-publish` since this is not only for tests, this is also for official build.